### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by alexsegura**
This runs OpenFisca + Mes Aides with Docker using Docker Compose. 
Just run `docker-compose up` to be up & running. 

**TODO**
- [x] Do not call "open" to open the browser

### CODE REVIEW

**Comment by alexsegura** [comment]
@guillett I think this is working fine now. 
Can we merge it in `master` so that I can stop rebasing my branches on this one? 🙏

File: Gruntfile.js [hunk 1]
```diff
@@ -338,14 +338,20 @@ module.exports = function (grunt) {
...
+    var tasks = [
+        'clean:server',
+        'concurrent:server',
+        'autoprefixer',
+        'express:dev',
+        'watch'
+    ];
+
+    // Insert "open" before "watch"
+    if (true !== grunt.option('no-open')) {
```
Comment (guillett) [inline Gruntfile.js]: Est-il possible de supprimer `open` de la liste plutôt de l'ajouter ?

Comment (alexsegura) [reply Gruntfile.js]: Ok.

File: Gruntfile.js [hunk 2]
```diff
@@ -338,14 +338,21 @@ module.exports = function (grunt) {
...
+        'clean:server',
+        'concurrent:server',
+        'autoprefixer',
+        'express:dev',
+        'open',
+        'watch'
+    ];
+
+    // Remove "open" task
+    if (true === grunt.option('no-open')) {
```
Comment (guillett) [inline Gruntfile.js]: `if (true === grunt.option('no-open')) {` devrait pouvoir être remplacé par `if (grunt.option('no-open')) {` ?

**Pull request merged**